examples/interactive_asr/vad.py

```diff
@@ -19,18 +19,16 @@
 
 from collections import deque
 
-import librosa
 import numpy as np
```

Comment:
Do you want the numpy dependency?

Reply:
We use it on line 225, but it'd be nice to get rid of it. I was mostly cleaning lint, but feel free to suggest a replacement :)

Reply:
Sure. It could be solved in a separate PR. Just notice it.